src/App.scss

```diff
@@ -1,39 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
 iframe {
   display: none;
 }
 
 html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
```

Comment:
Consider using a more common font stack to ensure better cross-browser compatibility. 🌐